src/components/content/heroes/aurora/weather-easter-egg/weather-api.test.ts

```diff
@@ -0,0 +1,134 @@
+import { WeatherType } from './weather-types';
+import {
+  convertTimeStringToTimestamp,
+  getWeatherDescription,
+} from './weather-api';
+import axios, { AxiosResponse } from 'axios';
+import { getSunTime } from './weather-api';
+
+const API_KEY = process.env.WEATHER_API_KEY;
```

Comment:
There is no difference. It had to be renamed so that the client can access it when it is deployed, so that the API can be used by the client when it is released.